platform/viewer/src/umdLoader.js

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
This is a bunch of stuff to load umd modules - however, if no umd modules are declared, then the basics are essentially just import("@ohif/..."), almost the same as the v3-stable code, so this is almost unchanged to v3-stable.